src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderBooking.kt

```diff
@@ -656,12 +656,6 @@ class BookingBuilder(
           offenderBooking = offenderBooking,
           date = date,
         )
-          .also {
-            offenderBooking.inOutStatus = "OUT"
-            offenderBooking.location = repository.lookupAgencyLocation("OUT")
-            offenderBooking.active = false
-            offenderBooking.bookingEndDate = date
-          }
```

Comment:
I previously added this without realising it was being done in the OffenderExternalMovement builder.